tests/manager/api/odl/test_api.py

```diff
@@ -421,6 +421,25 @@ def test_checkin_open_access(
             opds2_with_odl_api_fixture.patron, "pin", pool
         )
 
+    def test__notification_url(self):
+        short_name = "short_name"
```

Comment:
Since we go to the trouble of setting up the app context here, could we set our `short_name` to "cm", pull in the fixture, and use this same test to verify that our fixture's `_notification_url` stays in sync with the mainline version?